### PR TITLE
add forms page

### PR DESCRIPTION
**Pull request by tgenissel**

### CODE REVIEW

File: docs/forms.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# 🧪 Forms
+
+We recommand to use react-hook-form to make form on your application.
```
Comment (nanocom) [inline docs/forms.md]: We recommand to use react-hook-form to make form on your application.
=> We recomm**e**nd to use react-hook-form to make form**s** on your application.
=> We use react-hook-form to create forms.

File: docs/forms.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# Kiwi form components
+
+You can use @yousign/kiwi components with react-hook-form. Dedicated components for forms are already in @yousign/kiwi
+
+## Form validation
+
+To do form validation you can use react-hook-form and yup together.
+
+- [yup](https://github.com/jquense/yup)
```
Comment (nanocom) [inline docs/forms.md:52]: code example with yup ?

Comment (tgenissel) [reply docs/forms.md:52]: The link just under https://react-hook-form.com/get-started#SchemaValidation :)
An example directly from react-hook-form

Comment (nanocom) [reply docs/forms.md:52]: alright

**Pull request closed without merging**